native/src/main/java/org/ballerinax/asb/admin/Administrator.java

```diff
@@ -586,21 +719,26 @@ private static BMap<BString, Object> constructRulePropertiesArray(PagedIterable<
      * @param ruleName            name of the Rule
      * @return null
      */
-    public static Object deleteRule(BObject administratorClient, BString topicName,
+    public static Object deleteRule(Environment env, BObject administratorClient, BString topicName,
                                     BString subscriptionName, BString ruleName) {
         ServiceBusAdministrationClient clientEp = getAdminFromBObject(administratorClient);
```

Comment:
Lets change this `getAdminFromBObject` util name to be consistent with the new changes you did in sender + receiver clients

Reply:
Fixed with [c7b2b967805dc2c631ebe7f82b11447359d998a1](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/231/commits/c7b2b967805dc2c631ebe7f82b11447359d998a1)